tasks/custom_task/custom_task.py

```diff
@@ -47,6 +47,7 @@ def get_running_modes() -> list[str]:
         "pre_commit": os.environ.get("PRE_COMMIT", 0) == "1",
         "invoke_unit_tests": is_running_ut or os.environ.get("INVOKE_UNIT_TESTS", 0) == "1",
         "ci": running_in_ci(),
+        "pyapp": os.environ.get("PYAPP", 0) == "1",
```

Comment:
I know this is just to match the preceding lines but the return value will only ever be strings so we should change the default values here eventually.

Reply:
```suggestion
        "pyapp": os.environ.get("PYAPP", "") == "1",
```

Would this work?

Reply:
Yes that would be fine for all of them.

Reply:
Fixed. Thank you Ofek